caikit/runtime/dump_services.py

```diff
@@ -87,12 +87,15 @@ def dump_http_services(output_dir: str):
 
 
 if __name__ == "__main__":
-    assert len(sys.argv) == 2, f"Usage: {sys.argv[0]} <output_dir>"
+    assert (
+        len(sys.argv) == 2
+    ), f"Usage: {sys.argv[0]} <output_dir>, <write_modules_json>"
     out_dir = sys.argv[1]
+    write_modules_json = sys.argv[2]
```

Comment:
I think we might need some `bool` parsing on this since `python -m caikit.runtime.dump_services . false` would be "truthy". Realistically, this probably means adding `argparse` to this command. I think we would want  to preserve `output_dir` as a positional arg to avoid breaking compatibility.